jenkinsfiles/README.md

```diff
@@ -0,0 +1,17 @@
+## Mobile Application configurations
+
+Folder contains Jenkins pipeline definitions for mobile app builds. 
+Please refer to specific platform and architecture for more information.
+
+## Jenkinsfiles
+
+Files have groovy extension to support syntax higlighing.
+When moving to repositories they should be renamed to `Jenkinsfile`.
+
+- android_jenkinsfile.groovy
+- cordova_jenkinsfile.groovy
+
+## Development
+
+Pipeline files can be modifed for your own needs as long they would launch on the kubernetes node (docker slave).
```

Comment:
modified (spelling) 
Also should we describe how it would be launched - what makes it launch as a docker slave